docs/nulecule.md

```diff
@@ -0,0 +1,242 @@
+# Nulecule file
+
+A lot of information here is copied from the [Nulecule specification](https://github.com/projectatomic/nulecule/tree/master/spec).
+
+There are __minor__ differences as we implement features not yet implemented into `atomicapp` from the specification. Please refer to this reference for deploying your application.
```

Comment:
`There are __minor__ differences as we implement features not yet implemented into `atomicapp` from the specification.` 

I'm not sure exactly what you are trying to say with that sentence. Pretty sure it needs to be reworded